Resources/doc/config.rst

```diff
@@ -536,6 +569,10 @@ Full Default Configuration
                             prefix:               ~
                             alias:                ~
                             is_bundle:            ~
+            types:
+
+                # Prototype
+                name: Fully\Qualified\Class\Name
```

Comment:
I'd suggest using "custom_type" instead of "name" here, for consistency with the previous example. Likewise in the XML below.